src/labels/CustomLabels.labels

```diff
@@ -12691,4 +12691,124 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>stgValidationHHAccountHHRules</shortDescription>
         <value>The Account Model is set to &apos;Household Account&apos;, which requires Household Rules to be set to &apos;No Contacts&apos;.   When using Household Accounts, there is no need to have an additional Household Object.</value>
     </labels>
+     <labels>
+        <fullName>gsResourcesTitle</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The page title in the Getting Started Resources section</shortDescription>
+        <value>Resources</value>
+    </labels>
+    <labels>
+        <fullName>gsResourcesItemNonprofitCloudLabel</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Nonprofit Cloud label in Getting Started Resources section</shortDescription>
+        <value>Nonprofit Cloud Basics</value>
+    </labels>
+    <labels>
+        <fullName>gsResourcesItemNonprofitCloudLinkLabel</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Nonprofit Cloud link label in Getting Started Resources section</shortDescription>
+        <value>Get to Know Nonprofit Cloud</value>
+    </labels>
+    <labels>
+        <fullName>gsResourcesItemFundraisingLabel</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Fundraising label in Getting Started Resources section</shortDescription>
+        <value>Fundraising Learning Path</value>
+    </labels>
+    <labels>
+        <fullName>gsResourcesItemFundraisingLinkLabel</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Fundraising link label in Getting Started Resources section</shortDescription>
+        <value>Start Fundraising</value>
```

Comment:
@fparodi-salesforce I talked to Anne and Stephanie and we all agree that this should be changed to "Get Started" so that it has the same link text as the other Learning Path links. Can you please change it? Thanks!